ProjectSettings/ProjectSettings.asset

```diff
@@ -153,7 +153,7 @@ PlayerSettings:
   androidMaxAspectRatio: 2.1
   applicationIdentifier:
     Android: com.Company.ProductName
-    Standalone: Microsoft.HoloToolkit-Unity
+    Standalone: .HoloToolkit-MixedRealityToolkit
```

Comment:
Were these changed intentionally?

Reply:
Yeah it should probably just have the Holo replaced with MixedReality

Reply:
I changed it in the Editor.  Looks like Unity is still retaining the original project name in the asset.  I'll fix that.